packages/commonwealth/client/scripts/views/components/snapshot_proposal_selector/snapshot_proposal_selector_item.tsx

```diff
@@ -16,13 +17,13 @@ const SnapshotProposalSelectorItem = ({
   return (
     <div className="proposal-item" onClick={() => onClick(snapshot)}>
       <div className="selected">{isSelected && <CWCheck />}</div>
-      <div className="text">
-        <div className="proposal-item-text" title={snapshot.title}>
-          {snapshot.title}
-        </div>
-        <div className="proposal-item-subtext" title={snapshot.id}>
-          Hash: ${snapshot.id}
-        </div>
+      <div>
+        <CWText fontWeight="medium" noWrap title={snapshot.title}>
+          {smartTruncate(snapshot.title, 35)}
```

Comment:
CSS truncation did not work anymore? Or what is the reason to using js solution instead?

Reply:
we should use css truncation here! best saved for the css render step when possible